packages/taquito/src/prepare/prepare-provider.ts

```diff
@@ -1068,6 +1075,40 @@ export class PrepareProvider extends Provider implements PreparationProvider {
     };
   }
 
+  /**
+   *
+   * @description Method to prepare a failing_noop operation
+   * @param operation RPCOperation object or RPCOperation array
+   * @param source string or undefined source pkh
+   * @returns a PreparedOperation object
+   */
+  async failingNoop({ ...rest }: FailingNoopParams): Promise<PreparedOperation> {
+    const { pkh } = await this.getKeys();
+
+    const operation = await createFailingNoopOperation({
+      ...rest,
+    });
+
+    const ops = this.convertIntoArray(operation);
+
+    const hash = await this.getBlockHash(rest.basedOnBlock);
+    const protocol = await this.getProtocolHash();
+
+    this.#counters = {};
+    const headCounter = parseInt(await this.getHeadCounter(pkh), 10);
+
+    const contents = this.constructOpContents(ops, headCounter, pkh);
+
+    return {
+      opOb: {
```

Comment:
don't think it should prepared it as `PreparedOperation>` like it's going to be injected

Reply:
Isn't preparing a prerequisite of signing?

Reply:
failing_noop operation is only signing an object like this, will be good enough to prepared as is that will be ready to be signed. 
There is no counter, protocolhash needed needed, doesn't have to use `this.constructOpContents`
```
{
        branch: "B...",
        contents: [
          {
            kind: OpKind.FAILING_NOOP,
            arbitrary: "1234..."
          },
        ],
}
```